src/WorkflowCore/Services/BackgroundTasks/RunnablePoller.cs

```diff
@@ -51,27 +49,14 @@ private async void PollRunnables(object target)
         {
             try
             {
-                if (await _lockProvider.AcquireLock("poll runnables", new CancellationToken()))
+                _logger.LogInformation("Polling for runnable workflows");
+                var runnables = await _persistenceStore.GetRunnableInstances(_dateTimeProvider.Now);
+                foreach (var item in runnables)
                 {
-                    try
+                    if (await _lockProvider.AcquireLock($"wf:{item}", default))
```

Comment:
Do you think we should use a more specific lock key like `queue:wf:xxxx`?
ie. have different locks for queueing and processing?